### PR TITLE
[9.0][MIG][mass_mailing_partner]

### PR DESCRIPTION
**Pull request by archetipo**

### CODE REVIEW

File: mass_mailing_partner/README.rst [hunk 1]
```diff
@@ -37,7 +37,7 @@ Bug Tracker
 Bugs are tracked on `GitHub Issues <https://github.com/OCA/crm/issues>`_.
 In case of trouble, please check there if your issue has already been reported.
 If you spotted it first, help us smashing it by providing a detailed and welcomed feedback
-`here <https://github.com/OCA/crm/issues/new?body=module:%20mass_mailing_partner%0Aversion:%208.0%0A%0A**Steps%20to%20reproduce**%0A-%20...%0A%0A**Current%20behavior**%0A%0A**Expected%20behavior**>`_.
+`here <https://github.com/OCA/crm/issues/new?body=module:%20mass_mailing_partner%0Aversion:%209.0%0A%0A**Steps%20to%20reproduce**%0A-%20...%0A%0A**Current%20behavior**%0A%0A**Expected%20behavior**>`_.
```
Comment (pedrobaeza) [inline mass_mailing_partner/README.rst]: Use the new template that removes this part: https://raw.githubusercontent.com/OCA/maintainer-tools/master/template/module/README.rst

File: mass_mailing_partner/README.rst [hunk 2]
```diff
@@ -81,4 +82,4 @@ OCA, or the Odoo Community Association, is a nonprofit organization whose
 mission is to support the collaborative development of Odoo features and
 promote its widespread use.
 
-To contribute to this module, please visit http://odoo-community.org.
\ No newline at end of file
+To contribute to this module, please visit http://odoo-community.org.
```
Comment (pedrobaeza) [inline mass_mailing_partner/README.rst]: s/https/https

File: mass_mailing_partner/__openerp__.py [hunk 3]
```diff
@@ -3,7 +3,7 @@
 
 {
     "name": "Link partners with mass-mailing",
-    "version": "8.0.1.0.0",
+    "version": "9.0.1.0.0",
     "author": "Antiun Ingeniería S.L., "
               "Serv. Tecnol. Avanzados - Pedro M. Baeza, "
```
Comment (pedrobaeza) [inline mass_mailing_partner/__openerp__.py]: Please switch both Antiun & Serv. Tecnol... to `Tecnativa`, that is the new company

**Review by pedrobaeza — CHANGES_REQUESTED** [review]

**Comment by pedrobaeza** [comment]
Have you included it last 8.0 commits?

**Comment by archetipo** [comment]
@pedrobaeza  thanks , fixed and rebase with last 8.0 commit

File: mass_mailing_partner/README.rst [hunk 4]
```diff
@@ -64,9 +62,12 @@ Contributors
 ------------
 
 * Pedro M. Baeza <pedro.baeza@serviciosbaeza.com>
```
Comment (rafaelbn) [inline mass_mailing_partner/README.rst:64]: Please change to pedro.baeza@tecnativa.com

File: mass_mailing_partner/README.rst [hunk 5]
```diff
@@ -64,9 +62,12 @@ Contributors
...
-* Rafael Blasco <rafabn@antiun.com>
+* Rafael Blasco <rafael.blasco@tecnativa.com>
 * Antonio Espinosa <antonioea@antiun.com>
```
Comment (rafaelbn) [inline mass_mailing_partner/README.rst:66]: Please change to antonio.espinosa@tecnativa.com

**Review by rafaelbn — APPROVED** [review]
Minor changes commented.

**Review by rafaelbn — CHANGES_REQUESTED** [review]
I mistake, please close this PR and open it again in https://github.com/OCA/social
Thanks

**Comment by rafaelbn** [comment]
We have the same PR here: https://github.com/OCA/crm/pull/91

**Comment by pedrobaeza** [comment]
@archetipo, can you please move this to OCA/social? It was hosted here on 8.0 because OCA/social doesn't still lives, but now this place is more suitable.

**Comment by archetipo** [comment]
@pedrobaeza  @rafaelbn 
thanks, moved on OCA/social
see https://github.com/OCA/social/pull/113

**Pull request closed without merging**